src/routes/(console)/project-[region]-[project]/databases/database-[database]/(entity)/helpers/terminology.ts

```diff
@@ -0,0 +1,67 @@
+import type { Page } from '@sveltejs/kit';
+import { page as pageStage } from '$app/state';
+
+import { type Models } from '@appwrite.io/console';
+import { capitalize, plural } from '$lib/helpers/string';
+import type { Columns } from '$database/table-[table]/store';
+import type { Term, TerminologyResult, TerminologyShape } from '$database/(entity)/helpers/types';
+
+export type DatabaseType = 'legacy' | 'tablesdb' | 'documentsdb' | 'vectordb';
+export type Entity = Partial<Models.Table>;
+export type Field = Partial<Columns>;
+
+export const baseTerminology = {
+    tablesdb: {
+        entity: 'table',
+        field: 'column',
+        record: 'row'
+    },
+    documentsdb: {
+        entity: 'collection',
+        record: 'document'
```

Comment:
no columns/attributes here.
@ItzNotABug TODO: handle empty, undefined for `trackEvent` and `trackError` as per this.

Reply:
done.